package.json

```diff
@@ -51,6 +51,7 @@
     "i18next-browser-languagedetector": "^6.1.1",
     "i18next-http-backend": "^2.2.0",
     "ini": "^3.0.1",
+    "inversify": "^6.0.1",
```

Comment:
Do you think adding an extra dependency here( in the project ) would be beneficial, and if so, what are the advantages of using it?

Reply:
This is the package that allows us to do dependency injection. Do you have another suggestion of an approach to take? We could probably do a bunch of brute force stuff, to achieve the same end, but I'm not sure the ROI. What were your thoughts?

Reply:
yes, I agree